File: seoyeon/week1/귤 고르기.py
def solution(k, tangerine):
    answer = 0
    tangerine.sort()
    tmp1 = {}
    for i in tangerine:
        if i in tmp1 :
            tmp1[i] += 1
        else : 
            tmp1[i] = 1
    tmp2 = sorted(tmp1.items(), key = lambda x:x[1], reverse = True)
    bucket = 0
    if tmp2[0][1] >= k : return 1
    else :
        for i in tmp2 :
            if bucket < k :
                bucket += i[1]   
                answer += 1 
    return answer

File: yejee/week1/귤 고르기.java
class Solution {
    public int solution(int k, int[] tangerine) {
        int answer = 0;
        int max = 1; 
        for (int i = 0; i < tangerine.length; i++) {
            if (tangerine[i] > max) {
                max = tangerine[i]; // 현재 요소가 최댓값보다 크면 최댓값 업데이트
            }
        }
        int[] count = new int[max + 1]; // 크기가 i인 귤의 개수는 count[i]
        
        for (int i = 0; i < count.length; i++) { //count 배열 초기화
            count[i] = 0;
        }
            
        for( int i : tangerine){
            count[i]++;
        }
        for(int i = 0; i < count.length;i++){
            
        }
        
        
        return answer;
    }
}